.github/workflows/config.yml

```diff
@@ -0,0 +1,34 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉for opening your very first issue in this project. AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like! If you want to report a bug🐞 please follow our [REPORTING_GUIDELINES.md](https://github.com/anitab-org/open-source-programs-backend/blob/develop/REPORTING_GUIDELINES.md). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```

Comment:
@himanshu007-creator isn't their `Issue Template` instead of Reporting guidelines

Reply:
I went according to screen shot mentioned in the issue. I need to make issue template here right?

Reply:
You can find the issue template [here](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE)